### PR TITLE
Use `apiRequest` helper in GIBFT `submit`

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Leverage the `apiRequest` helper to make the `fetch` in GIBFT's `submit` function

## Testing done
Confirmed existing `submit` unit tests pass

## Acceptance criteria
- [x] Use `apiRequest` to in place of custom fetch code in the GIBFT `submit` helper and confirm functionality remains the same

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
Apologies for some of the formatting revisions that muddy the waters of this PR.

**Review by rianfowler — APPROVED** [review]

File: src/platform/utilities/environment/index.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
-  preview: { API_URL: 'https://staging-api.vets.gov', BASE_URL: 'https://preview.va.gov' },
-  devpreview: { API_URL: 'https://dev-api.vets.gov', BASE_URL: 'https://dev-preview.va.gov' },
-  development: { API_URL: (process.env.API_URL || 'https://dev-api.vets.gov'), BASE_URL: (process.env.BASE_URL || 'https://dev.vets.gov') },
-  local: { API_URL: `http://${location.hostname}:3000`, BASE_URL: `http://${location.hostname}:3001` },
-  e2e: { API_URL: `http://localhost:${process.env.API_PORT || 3000}`, BASE_URL: `http://localhost:${process.env.WEB_PORT || 3333}` }
+  production: {
+    API_URL: 'https://api.vets.gov',
+    BASE_URL: 'https://www.vets.gov'
+  },
+  staging: {
```
Comment (cehsu) [inline src/platform/utilities/environment/index.js]: Much clearer!

File: src/applications/edu-benefits/tests/feedback-tool/helpers.unit.spec.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ import {
 function setFetchResponse(stub, data) {
   const response = new Response();
   response.ok = true;
+  // So `apiRequest`'s `isJson` helper works. Should this mock be more robust?
+  response.headers.get = () => 'application/json';
```
Comment (cehsu) [inline src/applications/edu-benefits/tests/feedback-tool/helpers.unit.spec.js]: That sounds good.

**Review by cehsu — APPROVED** [review]
LGTM. It looks like the original ticket mentions other polling requests that we could convert. Would you mind setting up additional tickets for those?

**Comment by erikphansen** [comment]
@cehsu I created:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13337
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13336

**Pull request merged**